src/ddqa/widgets/input.py

```diff
@@ -23,6 +24,16 @@ class LabeledInput(HorizontalScroll):
     """
 
 
+class SwitchLabel(Label):
+    def __init__(self, label: str, switch: Switch) -> None:
+        super().__init__(label)
+        self.styles.text_style = Style(underline=True)
+        self.__switch = switch
+
+    def on_click(self):
+        self.__switch.action_toggle()
```

Comment:
Can't we just inline this in the `LabeledSwitch` class? E.g. adding the `on_click` method directly there and setting the style there as well?
Next best idea would probably to turn this into a more generic `ClickableLabel` taking a callback to execute on a click event rather than it being tied to a `Switch` widget necessarily.

Reply:
having a look

Reply:
Works fine, good idea thanks!